ethereum/executionlayer/src/test/java/tech/pegasys/teku/ethereum/executionlayer/client/auth/JwtSecretKeyLoaderTest.java

```diff
@@ -32,12 +32,26 @@
   void testGetSecretKey_ReadSecretFromProvidedFilePath(@TempDir final Path tempDir)
       throws IOException {
     final Path jwtSecretFile = tempDir.resolve(JwtSecretKeyLoader.JWT_SECRET_FILE_NAME);
+    final Path unprefixedJwtSecretFile =
+        tempDir.resolve(JwtSecretKeyLoader.UNPREFIXED_JWT_SECRET_FILE_NAME);
+
```

Comment:
Actually, probably worth splitting this into separate test methods.  So we test loading with prefix in one method, without prefix in one and with trailing whitespace in a third.